src/rdma.c

```diff
@@ -1184,11 +1184,17 @@ static int connRdmaConnect(connection *conn,
                            const char *addr,
                            int port,
                            const char *src_addr,
+                           int multipath,
                            ConnectionCallbackFunc connect_handler) {
     rdma_connection *rdma_conn = (rdma_connection *)conn;
     struct rdma_cm_id *cm_id;
     RdmaContext *ctx;
 
+    if (multipath) {
+        serverLog(LL_NOTICE, "RDMA: multipath not supported");
+        return C_ERR;
+    }
+
```

Comment:
Not familiar with RDMA.

But can  RDMA run alongside standard TCP ? If yes, maybe we don't want to throw an error and just ignore it.

Reply:
Yes, RDMA run alongside TCP. Currently there is no config to enable RDMA multipath, maybe assert here?

Reply:
Or ignore? If users want to enable RDMA and MPTCP.

Reply:
Done.